source/ambient.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module "node-fetch";
+declare module "parse-diff";
+declare module "lodash.includes";
+declare module "lodash.find";
+declare module "jest-runtime";
+declare module "jest-environment-node";
+
+declare module "*/package.json";
```

Comment:
https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#wildcard-character-in-module-names